apps/mark-scan/backend/schema.sql

```diff
@@ -7,6 +7,7 @@ create table election (
   is_test_mode boolean not null default true,
   polls_state text not null default "polls_closed_initial",
   ballots_printed_count integer not null default 0,
+  ballots_cast_since_last_ballot_box_change integer not null default 0,
```

Comment:
Any reason to not use the same schema as apps/scan here? (`ballot_count_when_ballot_bag_last_replaced`)? IIRC correctly, we landed on that because it only needed to be updated when the ballot bag actually gets replaced (rather than on every ballot), so there's a little less chance for something to go wrong.

Reply:
If I understand correctly, you're suggesting that we write `ballot_count_when_ballot_box_last_replaced` with the value of `ballots_printed_count` each time the box is emptied? This could work in practice but would be less accurate.

mark-scan allows voters to invalidate a ballot, so if 5 of 200 voters invalidate their ballot and end up printing 2 ballots each, there would be 200 ballots printed and 195 ballots in the box and 5 spoiled ballots elsewhere.

Since this approach would cause the ballot box to be emptied sooner rather than later, we don't risk overfilling the box, but we can't make any guarantees about the number of ballots in the box. I thought it might be a potential cause for alarm if we say the box limit is 200, a poll worker changes the box, and later they find only 195 ballots from that box change. With correct ballot accounting one would be able to attribute the other 5 to invalidated ballots.

Do you have a sense for how important it is that the number of ballots in the box at change time equals exactly the stated limit?